test/test_config.py

```diff
@@ -0,0 +1,49 @@
+import unittest
+
+import pytest
```

Comment:
Out of curiousity, what are the benefits of pytest versus plain unittest? I see the annotation below and that's clear, but are there additional benefits you're looking to use?